web/controllers/auth.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Notox.Auth do
+  import Plug.Conn
+
+  def init(_opts) do
```

Comment:
I'd consider having this return `opts`, just so that if `call` ever needs `opts` it's not surprising why they are `nil`. You can one line it: `def init(opts), do: opts`